star_realms/src/game/effects/actions.rs

```diff
@@ -142,9 +137,42 @@ pub fn get_action(name: &String) -> Option<Action> {
         }
     }
     match name.as_str() {
+        "goods" => Some(
```

Comment:
this is a great example where the `ActionConfig` really comes in handy
it makes a ton of things like this not special exceptions